neqo-http3/src/transaction_client.rs

```diff
@@ -122,15 +123,15 @@ enum TransactionSendState {
 
 //  This is used for normal request/responses.
 #[derive(Debug)]
-pub struct TransactionClient {
+pub(crate) struct TransactionClient {
     send_state: TransactionSendState,
     response_stream: ResponseStream,
     stream_id: u64,
     conn_events: Http3ClientEvents,
 }
 
 impl TransactionClient {
-    pub fn new(
+    pub(crate) fn new(
```

Comment:
If the type is `pub(crate)`, then I think that we should just use the undecorated `pub` for methods.